src/mame/upl/ninjakd2_v.cpp

```diff
@@ -339,13 +339,11 @@ static bool stencil_omegaf(   uint16_t pal ) { return( true ); }
 // This is very hackish.
 // (Is there a possibility that software can't select it but hardware can?)
 
-void ninjakd2_state::erase_sprites( bitmap_ind16 &bitmap)
+void ninjakd2_state::erase_sprites(bitmap_ind16 &bitmap)
 {
 	// if sprite overdraw is disabled, clear the sprite framebuffer
 	if (!m_next_sprite_overdraw_enabled)
-	{
 		m_sprites_bitmap.fill(0xf);
-	}
 	else
 	{
```

Comment:
Please don’t remove the braces from one path where all other paths have braces.  It’s a common recommendation to use braced compound statements for all paths if at least one path contains multiple lines.